pro_tes/middleware/task_distribution/distance.py

```diff
@@ -242,8 +247,10 @@ def calculate_distance(
                 distances_unique[ip_tuple] = distances_full["distances"][
                     ip_tuple
                 ]
-            except KeyError:
-                pass
+            except KeyError as exc:
+                raise KeyError(
+                     f"Distance not found for IP addresses: {ip_tuple}"
+                ) from exc
```

Comment:
"Distances not found for IP addresses" (distances in plural)

Reply:
Done